clustered/common/src/main/java/org/ehcache/clustered/common/messages/LifeCycleMessageFactory.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright Terracotta, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.ehcache.clustered.common.messages;
+
+import org.ehcache.clustered.common.ServerSideConfiguration;
+import org.ehcache.clustered.common.ServerStoreConfiguration;
+
+/**
+ *
+ */
+public class LifeCycleMessageFactory {
+
+  public EhcacheEntityMessage validateCacheManger(ServerSideConfiguration configuration){
+    return new LifecycleMessage.ValidateCacheManager(configuration);
+  }
+
+  public EhcacheEntityMessage configureCacheManager(ServerSideConfiguration configuration) {
```

Comment:
This should be `configureStoreManager` instead of `configureCacheManager` -- the cache manager is not a clustered concept.  (The same for the message class.)